### DISCUSSION TITLE
"contains" keyword not working in get_all_tweets

### DISCUSSION THREAD

**Discussion by PradeepNalluri** [cjbarrie/academictwitteR#240] [Q&A]
Hi, I would like to thank you for constructing this package it has been really helpful!

I am trying to get all the tweets that **contains** a certain key word 
the following is the code
`tweets <-
  get_all_tweets(
    query = "contains:filter1",
    start_tweets = "2018-12-31T00:00:00Z",
    end_tweets = "2021-01-01T00:00:00Z",
    lang = "en",
    n = Inf,
    data_path = "./store",
    bind_tweets = TRUE,
    export_query=TRUE)`
I am getting the following error 
```
Error in make_query(url = endpoint_url, params = params, bearer_token = bearer_token,  : 
  something went wrong. Status code: 400
```

I am really not sure if contains is supported or not or is there a way to get this. The main requirement is if I query for the word "test" I should get all the tweets containing _test, tests,testing, tested and so on_.  Thank you!

**Comment by chainsawriot** [comment]
@PradeepNalluri Don't invent search operators; especially when you haven't read the relevant [API documentation](https://developer.twitter.com/en/docs/twitter-api/tweets/search/integrate/build-a-query). You can make sure yourself whether or not "contains" is supported by reading the API documentation, which is a thing that you should do.

For questions related to how to bypass the tokenized match of the API, I think you can suggest the feature in [the official Twitter forum](https://twittercommunity.com/). For your use case, a simple -- but apparently stupid -- solution is to expand the word "test" to all its conjugations and join them all by OR operators.

**Reply by PradeepNalluri** [reply]
Hey @chainsawriot, thank you for responding. 
I exactly did not "invent" the filter _contains_ :grin: it is available in [enterprise API](https://developer.twitter.com/en/docs/twitter-api/enterprise/rules-and-filtering/building-a-rule), I should have mentioned. I doubted that it will work here since v2 API doesn't have it in the list, just wanted to check if there is a way out of the box instead of building a query with multiple ORs. Thanks again for letting me know that there isn't any other than what you had mentioned.
